laser_ruler:
  number_of_sensors: 8
  max_distance: 0.2        # [m]
  min_distance: 0.005      # [m]
  ruler_lenght: 0.01        # [m]
  distance_increment: 0.1  # [m] distance between sensors
imu:
  frame_id: "$(arg robot_name)/stm_imu_link"
  name: "stm_imu"
  acceleration_covariance: [0.005, 0, 0, 0, 0.005, 0, 0, 0, 0.005]
  velocity_covariance: [0.005, 0, 0, 0, 0.005, 0, 0, 0, 0.005]
  orientation_covariance: [-1.0, 0, 0, 0, -1.0, 0, 0, 0, -1.0]
  
  gyro_offset: [-0.0, -0.0, 0.0]
  acc_offset: [-0.0, -0.0, 0.0]
  mag_offset: [0.0, 0.0, 0.0]
  mag_scale: [0.0, 0.0, 0.0]
  # mag_offset: [-644.25, 236.25, 731.4]
  # mag_scale: [1.0112138016019716, 1.0112138016019716, 0.9783023366714355]

motor_contoller:
  max_command: 1000.0
  encoders_resolution: 6532
  joint:
    left:
      name: $(arg robot_name)/left_wheel_joint
    right:
      name: $(arg robot_name)/right_wheel_joint


pid:
  right:
    Kp: 22.0
    Ki: 27.0
    Kd: 0.1
    upperLimit: 1000.0
    lowerLimit: -1000.0
    alfa: 0.01
    windup: 1000.0
    timeout: 5
    downLimitOn: true
    upLimitOn: true
    windupOn: true
  left:
    Kp: 22.0
    Ki: 27.0
    Kd: 0.1
    upperLimit: 1000.0
    lowerLimit: -1000.0
    alfa: 0.01
    windup: 1000.0
    timeout: 5
    downLimitOn: true
    upLimitOn: true
    windupOn: true
    